app/build.gradle

```diff
@@ -36,6 +36,8 @@ android {
 
 dependencies {
 
+    implementation 'javax.inject:javax.inject:1'
```

Comment:
Added implementation 'javax.inject:javax.inject:1' to make the Inject annotation in the ListActivity.kt code functional, based on the recommendation from Android Studio.